histimator/estimator.py

```diff
@@ -1,9 +1,19 @@
+"""Built-in estimator classes.
+"""
+
+# from . import backend as K
+# if K.backend() == 'tensorflow':
+# import tensorflow as tf
+
+
 import numpy as np
```

Comment:
change this line into `import numpy` to avoid conflicts with the variable `np`

Reply:
ok `math` has been vetoed. 

np will be changed to nuis